packs/data/feats.db/mummy-dedication.json

```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>Whether through natural processes or foul rituals, your body was mummified, and you've risen as an undead mummy. You gain the mummy and undead traits, as well as basic undead benefits.</p>\n<p>Many effects that harm the living are far less effective against your desiccated form. Unfortunately, your dried-out body is also highly flammable. You gain the @Compendium[pf2e.feats-srd.Toughness]{Toughness} feat, but also fire weakness equal to half your level. Your fist draws moisture from the living by touch, making it more damaging and deadly. The damage die for your fist increases to 1d6 instead of 1d4, and your fist loses the nonlethal trait.</p>\n<p>Finally, you become supernaturally bound to the dominant terrain in which you were created, either because you formed naturally from the weather of that terrain or because of the local ingredients and methods used in an intentional mummification process. Choose arctic, desert, mountain, or swamp. This choice can't be changed and may alter the effects of some of your feats.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the mummy archetype.</p>"
+            "value": "<p>Whether through natural processes or foul rituals, your body was mummified, and you've risen as an undead mummy. You gain the mummy and undead traits, as well as basic undead benefits.</p>\n<p>Many effects that harm the living are far less effective against your desiccated form. Unfortunately, your dried-out body is also highly flammable. You gain the @Compendium[pf2e.feats-srd.Toughness]{Toughness} feat, but also fire weakness equal to half your level. Your fist draws moisture from the living by touch, making it more damaging and deadly. The damage die for your fist increases to 1d6 instead of 1d4, and your fist loses the nonlethal trait.</p>\n<p>Finally, you become supernaturally bound to the dominant terrain in which you were created, either because you formed naturally from the weather of that terrain or because of the local ingredients and methods used in an intentional mummification process. Choose arctic, desert, mountain, or swamp. This choice can't be changed and may alter the effects of some of your feats.</p>\n<p><strong>Special</strong> You can't select another dedication feat until you have gained two other feats from the @UUID[Compendium.pf2e.journals.vx5FGEG34AxI2dow.JournalEntryPage.ZewC2i5YdZPsWO8X#mummy]{Mummy} archetype.</p>"
```

Comment:
That link looks mangled, unless the UUID for mummy archetype ended up randomly ending with `#mummy` in which case I quit.

Reply:
Huh. Wonder how that happened?